File: PrimaMateria/PrimaMateria/Classes/XTRVersionChecker.m
//
// XTRVersionChecker.m
// PrimaMateria
//
//  Created by Jerry Porter on 3/22/2010.
//  Copyright (c) 2016 xTrensa. All rights reserved.
//

@implementation XTRVersionChecker

#pragma mark - Accessor Methods

+ (NSString *)appNameString {
    return [[NSBundle mainBundle] objectForInfoDictionaryKey: @"CFBundleDisplayName"];
}

+ (NSString *) appVersionString {
    return [[NSBundle mainBundle] objectForInfoDictionaryKey: @"CFBundleShortVersionString"];
}

+ (NSString *) copywriteString {
    return [[NSBundle mainBundle] objectForInfoDictionaryKey: @"CFBundleGetInfoString"];
}

@end

File: PrimaMateria/PrimaMateria/Classes/XTRVersionChecker.swift
//
//  XTRVersionChecker.swift
//  PrimaMateria
//
//  Created by Jerry Porter on 4/14/16.
//  Copyright ©2018 xTrensa. All rights reserved.
//

class XTRVersionChecker : NSObject {
    
    class var appNameString: String? {
        return Bundle.main.object(forInfoDictionaryKey: "CFBundleDisplayName") as? String
    }
    
    class var appVersionString: String? {
        return Bundle.main.object(forInfoDictionaryKey: "CFBundleShortVersionString") as? String
    }
    
    class var copywriteString: String? {
        return Bundle.main.object(forInfoDictionaryKey: "CFBundleGetInfoString") as? String
    }
    
}
